src/mixins/meta.ts

```diff
@@ -127,3 +155,51 @@ export function metaTag(
 
   return metaTagsObject;
 }
+
+// ALernate locales generator
+export interface AlterateLocalLink {
+  href: string;
+  hreflang: string;
+  rel: "alternate";
+}
+
+export interface OpenGraphMetaTag {
+  content: string;
+  property: string;
+}
+
+function pageAlternateLocales(locales: QMetaI18nAlternateLocale[], useOpengraph?: QMetaI18nAlternateLocale) {
+  const currentDomain = domain();
+  
+  const links: {[index: string]: AlterateLocalLink } = {};
+  const metaAlternate: {[index: string]: OpenGraphMetaTag} = {};
+
+  for (const locale of locales) {
```

Comment:
Conviene fare direttamente la destrutturazione, ti evita i vari `locale.locale` strani

`for (const { url, locale } of locales) {`

Reply:
You're right.